### PR TITLE
Add test for JSON marshaling for Teams

### PR DESCRIPTION
**Pull request by emiscode**
Resource covered: Teams

### CODE REVIEW

File: github/teams_test.go [hunk 1]
```diff
@@ -1585,3 +1585,89 @@ func TestTeamsService_CreateOrUpdateIDPGroupConnectionsBySlug_empty(t *testing.T
...
+			Description:  String("d"),
+			URL:          String("u"),
+			Slug:         String("s"),
+			Permission:   String("p"),
+			Privacy:      String("p"),
+			MembersCount: Int(1),
+			ReposCount:   Int(1),
+		},
+		LDAPDN:           String("l"),
+
```
Comment (gmlewis) [inline github/teams_test.go:1630]: Please delete this blank line and add one immediately before the `want` below.

File: github/teams_test.go [hunk 2]
```diff
@@ -1585,3 +1585,89 @@ func TestTeamsService_CreateOrUpdateIDPGroupConnectionsBySlug_empty(t *testing.T
...
+			"slug": "s",
+			"permission": "p",
+			"privacy": "p",
+			"members_count": 1,
+			"repos_count": 1
+		},
+		"ldap_dn": "l"
+		
+	}`
+	testJSONMarshal(t, u, want)
```
Comment (gmlewis) [inline github/teams_test.go:1672]: For consistency with the rest of the repo, please add a blank line before this one.

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @emiscode !
Just a couple minor tweaks, please.

**Review by gmlewis — APPROVED** [review]
Thank you, @emiscode !
LGTM.

In future PRs, please do not use force-push, as it makes it more difficult to review changes made since the last code review.  We always use squash-and-merge in this repo, so you can push as many commits as necessary, and they will be merged into one final commit.

Awiating second LGTM before merging.

**Review by gmlewis — CHANGES_REQUESTED** [review]
Whups, it looks like the code needs to be run through `gofmt` first, in order for the tests to pass.  Please fix.

**Review by gmlewis — APPROVED** [review]
Thank you, @emiscode !
LGTM.

Awaiting second LGTM before merging.

**Comment by emiscode** [comment]
Thank you @gmlewis!

I'm glad to be contributing to this project, and I really appreciated your care with details during your review process.

**Review by Parker77 — APPROVED** [review]
LGTM.

**Pull request merged**